src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/XamlToRtfWriter.cs

```diff
@@ -2753,14 +2750,12 @@ XamlToRtfError IXamlContentHandler.SkippedEntity(string name)
                 {
                     if (name.Length >= 4)
                     {
-                        string num = name.Substring(2, name.Length - 3);
+                        ReadOnlySpan<char> num = name.Substring(2, name.Length - 3);
```

Comment:
Did you mean `ReadOnlySpan<char> num = name.AsSpan(2, name.Length - 3);`?

Reply:
I did, yes, thanks. That will avoid another string.